packages/slice-machine/src/components/Breadcrumb/Breadcrumb.module.css

```diff
@@ -0,0 +1,23 @@
+.items {
+  display: flex;
+  gap: 4px;
+  list-style: none;
+  margin: 0px;
+  padding: 0px;
+}
+
+.item {
+  display: flex;
+  gap: 4px;
+  align-items: center;
+
+  &:last-child {
```

Comment:
Right, in trying to avoid using React.Children, I couldn't find another solution. CSS was the most straightforward way.

But I removed that and used React.Children instead. In this instance, I think it is a good use of it and shouldn't cause issues in the future. Now we don't render the separator in the DOM at all.